packages/ui-react/src/utils/location.ts

```diff
@@ -12,6 +12,6 @@ export const modalURLFromLocation = (location: Location, u: keyof AccountModals
 };
 
 // Create a full modal url including hostname, mostly for external use (e.g paypal successUrl)
-export const modalURLFromWindowLocation = (u: keyof AccountModals, queryParams?: QueryParamsArg) => {
+export const modalURLFromWindowLocation = (u: keyof AccountModals | null, queryParams?: QueryParamsArg) => {
```

Comment:
Is it still modal URL when the `u` param is not defined? Or should we have a separate function for this purpose?

Reply:
The goal of explicitly passing `null` to this function is to clear the modal parameter from the URL. We could create a separate util for this, but that would essentially do the same:

```tsx
export const closeModalURLFromWindowLocation = (queryParams?: QueryParamsArg) => {
  return createURL(window.location.href, { ...queryParams, u: null });
}
```

Alternatively, we could create a `createURLFromWindowLocation` util (following the convention) and empty the `u` param from the callee.

```tsx
const successUrlPaypal = offerType === 'svod' ? waitingUrl : createURLFromWindowLocation({ u: null });
```

Reply:
Or you can use the `createURLFromLocation` util (we do the same in the AccountModal useEffect):
```javascript
const successUrlPaypal = offerType === 'svod' ? waitingUrl : createURLFromLocation(location, { u: null });
```

Reply:
`createURL(window.location.href, { u: null })` looks good. We can have a small function for this. It is just a bit confusing to me now that we build a modal url without any modal.

Reply:
Updated